ontology/omiabis.md

```diff
@@ -10,11 +10,11 @@ domain: health
 homepage: https://github.com/OMIABIS/omiabis-dev
 products:
   - id: omiabis.owl
-tracker: https://github.com/OMIABIS/omiabis-dev/issues
 license:
   url: http://creativecommons.org/licenses/by/3.0/
   label: CC-BY
-activity_status: active
+is_obsolete: true
+activity_status: inactive
 ---
 
-OMIABIS is an open-source ontology of biobank administration. OMIABIS stands for Ontologized MIABIS where MIABIS is the Minimum Information About BIobank data Sharing.
+OMIABIS has been merged into OBIB (http://www.obofoundry.org/ontology/obib.html).
```

Comment:
Can we make a structured field for tracking when one ontology is merged into another

Reply:
Also if the repo is obsolete, GitHub has a way of "archiving" the repo as well. Should we ask that people do that (if there's even a contact person who could?)

Reply:
I think asking people to do that is a good idea! @mbrochhausen would you consider trying to archive the OMIABiS repo?

Reply:
Instructions on how to archive a repo: https://github.blog/2017-11-08-archiving-repositories/